src/main/java/io/cdap/plugin/switchcase/route/Router.java

```diff
@@ -97,6 +97,8 @@ public void transform(StructuredRecord input, MultiOutputEmitter<StructuredRecor
       } else {
         emitNullValue(input, emitter);
       }
+      // this is to handle skip default port handling
```

Comment:
this comment is not correct, it' s not for skipping default port handling.  default value and null value was already handled.

Reply:
I think you don' t need to add any comment here, `return` here should be obvious logic.

Reply:
removed the comment